python/source/state_representation/bind_cartesian_space.cpp

```diff
@@ -212,17 +213,23 @@ void cartesian_pose(py::module_& m) {
   }
 
   c.def(py::self *= py::self);
+  c.def("__imul__", [](const CartesianPose& self, const CartesianTwist& other) -> void { throw py::type_error("unsupported operand type(s) for *=: 'state_representation.CartesianPose' and 'state_representation.CartesianTwist'"); });
```

Comment:
I just want to check my understanding. In the Pose binding, you define `self *= self`, and then explicitly forbid the `__imul__` operator for derived cases. However, in the other derived classes (e.g. Twist) you do not bind `__imul__` to anything. And yet, the test case for e.g. `twist *= wrench` expects a TypeError which is apparently satisfied. 

I would imagine that, because we don't bind `self *= self` at all for the other derived classes, then `__imul__` is already undefined and so raises a type error. It's just a bit confusing at first.

Reply:
Yes, I believe this works as you describe it indeed.